examples/analytics/dist/index.html

```diff
@@ -33,19 +32,16 @@
     ])
   ]
 };
-</pre><h2>main.js</h2><pre>
-console.log('Test!');
-</pre><h2>partials/body.html</h2><pre>
-&#x3C;h1&#x3E;Hello world!&#x3C;/h1&#x3E;
+</pre><h2>main.js</h2><pre>console.log('Test!');
+</pre><h2>partials/body.html</h2><pre>&lt;h1&gt;Hello world!&lt;/h1&gt;
 ...and the rest of this configuration example
-</pre><h2>partials/head.html</h2><pre>
-&#x3C;!-- Global site tag (gtag.js) - Google Analytics --&#x3E;
-&#x3C;script async src=&#x22;https://www.googletagmanager.com/gtag/js?id=&#x3C;%= ga_property_id %&#x3E;&#x22;&#x3E;&#x3C;/script&#x3E;
-&#x3C;script&#x3E;
+</pre><h2>partials/head.html</h2><pre>&lt;!-- Global site tag (gtag.js) - Google Analytics --&gt;
+&lt;script async src="https://www.googletagmanager.com/gtag/js?id=&lt;%= ga_property_id %&gt;"&gt;&lt;/script&gt;
+&lt;script&gt;
 window.dataLayer = window.dataLayer || [];
 function gtag(){dataLayer.push(arguments);}
-gtag(&#x27;js&#x27;, new Date());
+gtag('js', new Date());
```

Comment:
for what it's worth, i checked the example in the browser and these seem to be fine - i guess only the < and > need to be escaped

Reply:
I don't understand exactly what you are trying to say here about escaping. Can you please elaborate. Even I am confused about the escaping and serialisation. The DOMParser might export ugly HTML code but I think it looks just fine when we look at it in the browser, which is the goal.

Reply:
yeah i think we're good :) im just talking out loud